lib/logflare_web/router.ex

```diff
@@ -308,6 +311,8 @@ defmodule LogflareWeb.Router do
 
     delete("/accounts/:id", AdminController, :delete_account)
     get("/accounts/:id/become", AdminController, :become_account)
+
+    live_dashboard("/livedashboard", ecto_repos: [], metrics: Logflare.Telemetry)
```

Comment:
should scope it under `/admin`

Reply:
it's already under admin
<img width="725" alt="Screenshot 2023-06-19 at 21 51 17" src="https://github.com/Logflare/logflare/assets/1697301/4978faf1-2789-4d22-95eb-95f8eb8c5d46">

Reply:
Ah oops by bad 👍👍